mix.exs

```diff
@@ -125,6 +126,54 @@ defmodule Phoenix.Mixfile do
     ]
   end
 
+  defp groups_for_modules do
+    # Ungrouped Modules:
+    #
+    # Phoenix
+    # Phoenix.Channel
+    # Phoenix.Controller
+    # Phoenix.Logger
```

Comment:
What do you think about this:

Have an "Endpoint and Plugs" category, move Phoenix.CodeReloader and Phoenix.Logger in there. Rename the "Development and Testing" to testing.

The reason for this is because Phoenix.Logger is really not at the same level as these.

Thoughts?

Reply:
I don't think it would be immediately obvious that they would be in the "Endpoint and Plugs" section without prior knowledge that those modules are plugs. However, I think it is very likely that the users who want the docs for those modules will have that prior knowledge.

Reply:
Well, to help with that, those two modules are used in the endpoint as `plug Plug.CodeReloader` and `plug Phoenix.Logger`. So I think they do fit all together. :)